packages/blockchain-wallet-v4-frontend/src/modals/Staking/AccountSummary/AccountSummary.detail.template.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactNode } from 'react'
+
+import { Icon, Text, TooltipHost } from 'blockchain-info-components'
+
+import {
+  DetailsContainer,
+  LeftContainer,
+  RightContainer,
+  SubTextContainer
+} from './AccountSummary.model'
+
+const Details = ({ subText, subValue, text, tooltipId, value }: DetailsType) => (
+  <DetailsContainer>
+    <LeftContainer>
+      <Text color='grey900' size='14px' weight={600}>
+        {text}
+      </Text>
+      {subText && (
+        <SubTextContainer>
+          <Text color='grey700' size='14px' weight={500}>
+            {subText}
+          </Text>
+          {tooltipId && (
+            <TooltipHost id={tooltipId}>
+              <Icon name='info' size='12px' color='grey400' />
```

Comment:
Not a big deal but we should start using constellation components as much as possible from now on. With the eventual goal of deleting the `blockchain-info-components` repo. Especially for icons like the one you use here: https://blockchain.github.io/blockchain-wallet-v4-frontend/?path=/story/constellation_base-icon--icon&args=icon:IconInformation